meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/start-resin-supervisor

```diff
@@ -76,7 +76,7 @@ runSupervisor() {
         --restart=always \
         --net=host \
         --cidenv=SUPERVISOR_CONTAINER_ID \
-        -v /var/run/balena-engine.sock:/var/run/balena-engine.sock \
+	--mount type=bind,source=/var/run/balena-engine.sock,target=/var/run/balena-engine.sock \
```

Comment:
bad indentation?

Reply:
ugh. We have a mix of space/tabs overall which is annoying. looked ok in vim

will fix up